### PR TITLE
Add verify-docs job

### PR DESCRIPTION
**Pull request by geyslan**
### 1. Explain what the PR does

fdee7e01a **chore(ci): add verify-docs job**

### CODE REVIEW

**Comment by geyslan** [comment]
It seems to work, the [commit](https://github.com/aquasecurity/tracee/pull/3672/commits/b2f8c940b37bc5686e793851fd3ac6631150f4ee) triggered the [error](https://github.com/aquasecurity/tracee/actions/runs/6747959665/job/18345245719?pr=3672):

image

**Comment by geyslan** [comment]
Cancelled jobs since the one related to this PR passed: https://github.com/aquasecurity/tracee/pull/3672/checks

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -67,6 +69,49 @@ env:
...
+            man_files:
+              - "docs/man/**/*.1"
+      - name: Ensure updates of *.1.md and *.1 pairs
+        if: ${{ steps.changed-files.outputs.md_files_any_changed == 'true' || steps.changed-files.outputs.man_files_any_changed == 'true' }}
+        run: |
+          md_files=$(echo "${{ steps.changed-files.outputs.md_files_all_changed_files }}" | xargs -n 1 basename | sed 's/\.1.md$//')
+          man_files=$(echo "${{ steps.changed-files.outputs.man_files_all_changed_files }}" | xargs -n 1 basename | sed 's/\.1$//')
+
+          missing_updates=()
+          for base_name in $md_files; do
```
Comment (rafaeldtinoco) [inline .github/workflows/pr.yaml:97]: The script assumes that for every .1.md file, there is a corresponding .1 file that needs to be updated, and vice versa. If this is not strict (like a new file added), the script may raise false positives, no ?

Comment (geyslan) [reply .github/workflows/pr.yaml:97]: Well observed. I need to change it to [any_modified](https://github.com/tj-actions/changed-files#output_any_modified) in the `if` and to [all_modified_files](https://github.com/tj-actions/changed-files#output_all_modified_files) in the assignments. Going to, hold on.

Comment (geyslan) [reply .github/workflows/pr.yaml:97]: Updated Makefile to ensure that any change (addition, modification, deletion) in the .md will reflect in the man pages.

Comment (geyslan) [reply .github/workflows/pr.yaml:97]: image

This error should be ignored, as it was a remnant caused precisely by the lack of checking (checking introduced by this PR).

**Review by rafaeldtinoco — APPROVED** [review]
LGTM

**Pull request merged**